openassessment/workflow/models.py

```diff
@@ -660,12 +677,12 @@ def update(self, submission_uuid, assessment_requirements):
         assessment_finished = getattr(self.api(), 'assessment_is_finished', default_finished)
```

Comment:
Correct - a given instance of `AssessmentWorkflowStep` can correspond to any of a number of `api()`-s (self, peer, staff, etc), and those apis are not forced to define these attributes, so we use the `lambda => True` default in those cases.